### ISSUE TITLE
[Question] Difference between `quiche::Error:FinalSize` stream flow control capacity and `conn.stream_capcity()`

### ISSUE THREAD

**Issue by mickel8** [cloudflare/quiche#871]
Hi,
I am trying to modify client example from `examples` section so that it can send two requests in a row.
When trying to send the second request I get `quiche:Error::FinalSize` and I can't get what does it mean. 

I can't also get the difference between `quiche::Error:FinalSize`, `conn.stream_capacity()` and `config.set_initial_max_data()`. 
The last one is for setting stream flow control but what is `stream_capacity()`? Is it something like stream buffer size i.e. max data we can write to stream? If yes why my stream has `~13600` capacity but when trying to write I get `FinalSize`. 

Thanks in advance

**Comment by ghedo** [comment]
Yes, sorry for the delay, `FinalSize` likely meant that you called `stream_send()` with `fin=true` multiple times.
